frontend/src/routes/Home/Welcome/Welcome.tsx

```diff
@@ -0,0 +1,158 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { Card, CardBody, CardTitle, Page, PageSection, Stack, Text, Title } from '@patternfly/react-core'
+import { useMemo } from 'react'
+import { Link } from 'react-router-dom'
+import { AcmMasonry } from '../../../components/AcmMasonry'
+import { NavigationPath } from '../../../NavigationPath'
+
+export default function WelcomePage() {
+    const cards = useMemo(
+        () => [
+            {
+                title: 'End-to-end visibility',
+                description:
+                    'Go to Overview View system alerts, critical application metrics, and overall system health. Search, identify, and resolve issues that are impacting distributed workloads using an operational dashboard designed for Site Reliability Engineers (SREs).',
+                link: NavigationPath.overview,
+            },
+            {
+                title: 'Cluster lifecycle',
+                description:
+                    'Create, update, scale, and remove clusters reliably, consistently using an open source programming model that supports and encourages Infrastructure as Code best practices and design principles.',
+                link: NavigationPath.clusters,
+            },
+            {
+                title: 'Application lifecycle',
+                description:
+                    'Define a business application using open standards and deploy the applications using placement policies that are integrated into existing CI/CD pipelines and governance controls.',
+                link: NavigationPath.applications,
+            },
+            {
+                title: 'Governance, Risk, and Compliance',
+                description:
+                    'Use policies to automatically configure and maintain consistency of security controls required by industry or other corporate standards. Prevent unintentional or malicious configuration drift that might expose unwanted and unnecessary threat vectors.',
+                link: NavigationPath.governance,
+            },
+            {
+                title: 'Multicluster networking',
+                description:
+                    'Enable direct networking connection between different on-premises or cloud-hosted Kubernetes clusters by grouping them in cluster sets and enabling the Submariner add-on.',
+                link: NavigationPath.overview,
+            },
+        ],
+        []
+    )
+
+    const details = useMemo(
```

Comment:
nit: There is no need for the `useMemo`, the object can be simply one-time created, i.e. in the top-level scope of this file.

Reply:
that's true if `const details = [...]` is part of the `WelcomePage` function. If created outside of the WelcomePage (on the same), it will be created just once. There is no need to generate the array - it's all static if I am not mistaken.

Reply:
In general I try to avoid moving variables up to the top-level scope.